src/kcas.mli

```diff
@@ -426,14 +432,14 @@ module Xt : sig
       to be the given value [v] in the explicit transaction log [xt]. *)
 
   val update : xt:'x t -> 'a Loc.t -> ('a -> 'a) -> 'a
-  (** [update ~xt r f] is equivalent to [let x = get ~xt r in set ~xt r (f x); x]
-      with the limitation that [f] must not and is not allowed to record
-      accesses to the transaction log. *)
+  (** [update ~xt r f] is equivalent to [let x = get ~xt r in set ~xt r (f x);
+      x] with the limitation that [f] must not and is not allowed to access the
+      transaction log. *)
```

Comment:
The comment here was reworded to avoid the impression that `is_in_log` would be allowed.